### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by martin226**
**Pull Request Description**

Minor typo fixed ("Todays" -> "Today's")

### CODE REVIEW

**Review by kmiu2 — APPROVED** [review]
nice

**Comment by kmiu2** [comment]
@martin226 all checks passed, feel free to merge it when you want

**Comment by martin226** [comment]
Seems like I don't have write access so I can't merge the PR

**Pull request merged**
